.github/workflows/container-image.yml

```diff
@@ -61,6 +86,70 @@ jobs:
       with:
         image: "samba-ad-server:latest"
         retention_days: 1
+  build--centos-client:
+    runs-on: ubuntu-latest
+    env:
+      BUILDAH_FORMAT: oci
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build the centos client image
+      run: make OS_NAME=centos build-client
+    # Here we upload samba-client image to artifacts locally for consumption
+    # during samba-toolbox build process.
+    - name: Upload client image
+      uses: ishworkh/docker-image-artifact-upload@v1
+      with:
+        image: "quay.io/samba.org/samba-client:icentos-latest"
+        retention_days: 1
+
+  build-centos-toolbox:
+    needs: build-centos-client
+    runs-on: ubuntu-latest
+    env:
+      BUILDAH_FORMAT: oci
+    steps:
+    - uses: actions/checkout@v3
+    # Download locally stored samba-client image to be used as base for building
+    # samba-toolbox.
+    - name: Download client image
+      uses: ishworkh/docker-image-artifact-download@v1
+      with:
+        image: "quay.io/samba.org/samba-client:centos-latest"
+    - name: Build the toolbox image
+      run: make OS_NAME=centos build-toolbox
+
+
+  build-client:
+    runs-on: ubuntu-latest
+    env:
+      BUILDAH_FORMAT: oci
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build the client image
+      run: make build-client
+    # Here we upload samba-client image to artifacts locally for consumption
+    # during samba-toolbox build process.
+    - name: Upload client image
+      uses: ishworkh/docker-image-artifact-upload@v1
+      with:
+        image: "quay.io/samba.org/samba-client:latest"
+        retention_days: 1
+
+  build-toolbox:
+    needs: build-client
+    runs-on: ubuntu-latest
+    env:
+      BUILDAH_FORMAT: oci
+    steps:
+    - uses: actions/checkout@v3
+    # Download locally stored samba-client image to be used as base for building
+    # samba-toolbox.
+    - name: Download client image
+      uses: ishworkh/docker-image-artifact-download@v1
+      with:
+        image: "quay.io/samba.org/samba-client:latest"
+    - name: Build the toolbox image
+      run: make build-toolbox
```

Comment:
`build-client` and `build-toolbox` are already existing jobs. This is duplication.

Reply:
not sure how this ended up in the commit. I must have made a mistake adjusting the copied jobs to centos ... Will try to fix. thanks for spotting this.

Reply:
I can still see the duplication and this should be the reason for github actions not getting triggered.
```
Invalid workflow file: .github/workflows/container-image.yml#L154
You have an error in your yaml syntax on line 154
```